ls1intum/Artemis
# Bugfix: K6 SCA & Exam

### Checklist

### Motivation and Context
There were some issues with the K6 tests related to SCA and Exams (new constraints that were not checked before).

### Description
Adjusted the tests to ensure the constraints are fullfilled.

### Steps for Testing
Verify that the tests pass on Bamboo without error.

https://bamboo.ase.in.tum.de/browse/ARTEMIS-SAT6-6
https://bamboo.ase.in.tum.de/browse/ARTEMIS-SATJG6-7
